pkgs/desktops/lxqt/lxqt-build-tools/default.nix

```
@@ -21,6 +22,13 @@ mkDerivation rec {
     sha256 = "0zhcv6cbdn9fr5lpglz26gzssbxkpi824sgc0g7w3hh1z6nqqf8l";
   };
 
+  # Nix clang on darwin identifies as 'Clang', not 'AppleClang'
+  # Without this, dependants fail to link.
+  postPatch = lib.optionalString stdenv.isDarwin ''
+    substituteInPlace cmake/modules/LXQtCompilerSettings.cmake \
+      --replace AppleClang Clang
+  '';
```

```suggestion
  postPatch = ''
    substituteInPlace cmake/modules/LXQtCompilerSettings.cmake \
      --replace AppleClang Clang
  '';
```

Otherwise we might miss it when updating on Linux.

---

Updated.

I'm now a little worried in general that this kind of substitution may break something when upstream these conditions in the cmake code. Then again, that'll probably result in a broken Darwin build, or even a broken build everywhere. So I guess it's fine?